### PR TITLE
Create percentual bucketspan vars

### PR DESCRIPTION
**Pull request by albertoperdomo2**
In some cases scenarios, the status panel is used to go from one dashboard to a more specific one, e.g. when performing root/cause analysis. In such cases, the user may want to reduce the scope of the time span, while keeping the desired event centered (to be able to analyse the previous/posterior buckets of time). 

The status panel does not have dynamically arranged variables to do so, and this is what this PR tries to solve. It is a super-duper simple implementation which comes from the experience. After some tests, we found that 4h bucket span works perfectly with a 7 days dashboard time range, which is a 2.4% of it. 

This way, the scope and time span of the dashboard can be reduced, acting like a "zoom".

### CODE REVIEW

**Comment by diafour** [comment]
Hello! Thank you for PR!

Can you add the description of new variables in README.md?

**Comment by albertoperdomo2** [comment]
@diafour done!

File: README.md [hunk 1]
```diff
@@ -194,6 +194,17 @@ __Items__ is a list of definitions to display URLs in tooltip.
...
+In some cases scenarios, the status panel is used to go from one dashboard to a more specific one, e.g. when performing root/cause analysis. In such cases, the user may want to reduce the scope of the time span, while keeping the desired event centered (to be able to analyse the previous/posterior buckets of time). Those values are introduced through `__bucket_from` and `__bucket_to`. 
+
+`__bucket_from`: It's the value of `bucket.from` minus the percentual bucket.
+`__bucket_to`: It's the value of `bucket.to` plus the percentual bucket.
+
+## Development
+
+To test and improve the plugin you can run Grafana instance in Docker using following command (in
+the directory containing Statusmap plugin):
+
```
Comment (diafour) [inline README.md:207]: Something is missing here. Also, there is a DEVELOPMENT.md file with commands for the plugin developer.

Comment (albertoperdomo2) [reply README.md]: I'm sorry, the PR was quite behind master. I'm closing this PR and opened a new one in #242. Sorry for the inconveniences, let's continue there.

**Comment by albertoperdomo2** [comment]
The content of this PR is now in #242

**Pull request closed without merging**